### ISSUE TITLE
[ERROR] Can't login to microsoft account

### ISSUE THREAD

**Issue by ghost** [BleachDrinker420/BleachHack#515]
I Can't login to my microsoft account on bleachhack.
Error >> Failed to find MS redirect link.
I'm using linux so if that's a problem, let me know

**Comment by TangyKiwi** [comment]
Alt manager has been broken since 1.19 iirc, not an issue on your side.

**Comment by ghost** [comment]
> Alt manager has been broken since 1.19 iirc, not an issue on your side.

yeah, but on my windows pc i can login to Microsoft using bleachhack

**Comment by g07826329423** [comment]
It seems like the issue may be related to how the Microsoft account authentication process handles redirects, especially on Linux. Make sure that your network settings allow for the necessary callbacks, and consider checking if there’s an alternative method for authentication specified in the BleachHack documentation. Additionally, ensure that all dependencies are correctly installed, as this could also affect the redirect handling.
